templates/propertyGroup.html

```diff
@@ -1,6 +1,7 @@
 <h3 class="propertyGroup__title">
 	<i ng-class="ctrl.getIconClass()"></i>
 	{{ctrl.getReadableName()}}
+	<i ng-class="ctrl.getInfoClass()" title="{{ctrl.getInfoText()}}"></i>
```

Comment:
We also should only add this element if there is a text, use ng-if and the length of the getInfoText? :)